src/neptune/new/handler.py

```diff
@@ -287,6 +300,102 @@ def log(
                 self._container.set_attribute(self._path, attr)
             attr.log(value, step=step, timestamp=timestamp, wait=wait, **kwargs)
 
+    def append(
+        self,
+        value: Union[dict, Any],
```

Comment:
I don't think it makes any sense to define `Union` of `Any` with anything else. `dict` is also an `Any`.

Reply:
I wanted to address this problem with our custom well-defined types based on agreement with the product.
The expected type is nested structure with collections in the leaf.

Before it's implemented I wanted to somehow `mock` this place with such type annotation.
I agree it looks lame. If you want to I can replace it with Any and add comment what Any is going to be.